src/Cake.Common/Build/BuildSystemAliases.cs

```diff
@@ -126,5 +129,27 @@ public static IBambooProvider Bamboo(this ICakeContext context)
             var buildSystem = context.BuildSystem();
             return buildSystem.Bamboo;
         }
+
+        /// <summary>
+        /// Gets a <see cref="Cake.Common.Build.ContinuaCI.ContinuaCIProvider"/> instance that can
+        /// be used to manipulate the Continua CI environment.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// var isContinuaCIBuild = ContinuaCI.IsRunningContinuaCI;
+        /// </code>
+        /// </example>
+        /// <param name="context">The context.</param>
+        /// <returns>A <see cref="Cake.Common.Build.ContinuaCI"/> instance.</returns>
+        [CakePropertyAlias(Cache = true)]
+        public static IContinuaCIProvider ContinuaCI(this ICakeContext context)
```

Comment:
Since the namespaces for `ContinuaCIMessageType` isn't imported, we can tell Cake to import them by adding the namespaces that should be imported with the `ContinuaCI` alias:

```csharp
[CakeNamespaceImport("Cake.Common.Build.ContinuaCI")]
[CakeNamespaceImport("Cake.Common.Build.ContinuaCI.Data")]
```